rosbag2_tests/test/rosbag2_tests/test_rosbag2_record_end_to_end.cpp

```diff
@@ -564,7 +564,7 @@ TEST_F(RecordFixture, record_fails_if_both_all_and_topic_list_is_specified) {
   auto error_output = internal::GetCapturedStderr();
 
   EXPECT_THAT(exit_code, Eq(EXIT_FAILURE));
-  EXPECT_THAT(error_output, HasSubstr("Can not specify topics and -a at the same time."));
+  EXPECT_THAT(error_output, HasSubstr("Must specify only one option out of topics, regex or -a"));
```

Comment:
Checking the string exactly isn't very useful, it's just a ["change detector"](https://testing.googleblog.com/2015/01/testing-on-toilet-change-detector-tests.html) - this should probably just check that the error output isn't empty